docs/start/getting-started/fragments/vanillajs/setup.md

```diff
@@ -123,6 +156,8 @@ module.exports = {
 };
 ```
 
+By default `webpack-dev-server` will start on port `8080`. You may need to change devServer host to `0.0.0.0` if you are running your web-server in a Cloud IDE such as AWS Cloud9.
```

Comment:
I think this is too much detail for a getting started. My thought: If people are aware of Cloud IDEs they probably will know how to setup webpack. @sammartinez - thoughts?

Reply:
Chatted with Sam offline. This seems fine. I'll resolve this comment.

Reply:
This should be fine, but we may want to recommend locking down to your IP so this isnt open to the world